overlord/snapshotstate/snapshotstate.go

```diff
@@ -97,6 +98,11 @@ func AutomaticSnapshotExpiration(st *state.State) (time.Duration, error) {
 		}
 		logger.Noticef("snapshots.automatic.retention cannot be parsed: %v", err)
 	}
+	// TODO: automatic snapshots are currently disable on Ubuntu Core
```

Comment:
s/disable/disabled by default/

Reply:
Fixed via force push (for easier cherry picking)